compiler-rt/lib/profile/InstrProfilingUtil.c

```diff
@@ -258,6 +267,120 @@ COMPILER_RT_VISIBILITY FILE *lprofOpenFileEx(const char *ProfileName) {
   return f;
 }
 
+// Return 1 (true) if the file descriptor Fd represents a file that is on a
+// local filesystem, otherwise return 0.
+COMPILER_RT_UNUSED static int is_local_filesystem(int Fd) {
```

Comment:
Can we avoid using a mixture of `snake_case` and `camelCase` for function names (i.e. `is_local_filesystem` vs `isMmapSafe`)?

Reply:
done